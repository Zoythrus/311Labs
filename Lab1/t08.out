x is: 64 and digit is: 4
x is: 40 and digit is: 0
x is: 97 and digit is: 7
x is: 73 and digit is: 3
x is: 95 and digit is: 5
x is: 66 and digit is: 6
x is: 42 and digit is: 2
x is: 29 and digit is: 9
x is: 17 and digit is: 7
x is: 70 and digit is: 0
bins[k][l] is: 0
bins[k][l] is: 0
bins[k][l] is: 0
bins[k][l] is: 0
bins[k][l] is: 0
bins[k][l] is: 0
bins[k][l] is: 0
bins[k][l] is: 0
bins[k][l] is: 0
bins[k][l] is: 0
x is: 2 and digit is: 2
x is: 1 and digit is: 1
x is: 9 and digit is: 9
x is: 7 and digit is: 7
x is: 9 and digit is: 9
x is: 6 and digit is: 6
x is: 4 and digit is: 4
x is: 2 and digit is: 2
x is: 1 and digit is: 1
x is: 7 and digit is: 7
bins[k][l] is: 0
bins[k][l] is: 0
bins[k][l] is: 0
bins[k][l] is: 0
bins[k][l] is: 0
bins[k][l] is: 0
bins[k][l] is: 0
bins[k][l] is: 0
bins[k][l] is: 0
bins[k][l] is: 0


97
95
97
73
95
66
42
29
17
70
