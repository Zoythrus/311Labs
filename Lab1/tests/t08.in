radix
10 2
64
40
97
73
95
66
42
29
17
70
