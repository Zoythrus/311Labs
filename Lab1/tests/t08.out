17
29
40
42
64
66
70
73
95
97
